pyworkflow/plugin.py

```diff
@@ -359,7 +359,7 @@ def printInfo(cls):
         print("     wizards: %s" % len(cls._wizards))
 
     # ---------- Private methods of Domain class ------------------------------
-    @classmethod
+    @staticmethod
     def __pluginNotFound(cls, plugName, errorMsg='', doRaise=False):
```

Comment:
staticmethod does not receive cls as first argument

Reply:
upss!